.golangci.yml

```
@@ -57,6 +58,14 @@ issues:
 linters-settings:
   gofmt:
     simplify: true
+  forbidigo:
+    # Forbid the following identifiers (list of regexp).
+    forbid:
+      - '\brequire\.New\b'
+      - '\bassert\.New\b'
```

```suggestion
      - '\brequire\.New\b(# Use package-level functions instead)?'
      - '\bassert\.New\b(# Use package-level functions instead)?'
```

This will make the output like:
```
use of `ast.Inspect` forbidden because "use package-level functions instead" at blah.go:32:2
```

---

Oh merge happened already 😆 not a big deal